.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+| Q | A
+| --- | ---
+| Bug? | no
+| New Feature? | no
+| Version | Specific version or SHA of a commit
+| Sulu Version | Specific version or SHA of a commit
+| Browser Version | Browser name and version
+
+#### Actual Behavior
+
+How does Sulu behave at the moment? 
+
+#### Expected Behavior
+
+What is the behavior you expect?
+
+#### Steps to Reproduce
+
+What are the steps to reproduce this bug? Please add code examples,
+screenshots or links to GitHub repositories that reproduce the problem.
+
+#### Possible Solutions
+
+If you have already ideas how to solve the issue, add them here.
+(remove this section if not needed)
+
```

Comment:
Only one new line at the end

Reply:
its only one new line at the end? or do you mean no new lines?